apprentice/agents/ModularAgent.py

```diff
@@ -634,6 +664,7 @@ def train(self, state:Dict, sai:Sai=None, reward:float=None,
                     rhs_by_how = self.rhs_by_how.get(skill_label, {})
                     for exp in explanations:
                         # print("FOUND EX:", str(exp))
+                        agent_logger.error("FOUND EX:", str(exp))
```

Comment:
The logging library's various message functions don't automatically concatenate arguments in the same way that print does so it can't be used as a drop-in replacement like this. See: https://docs.python.org/3/library/logging.html#logging.Logger.debug. When I run this code on my end I mostly just see a bunch of exception stack traces.